aio/tools/transforms/angular-base-package/services/getPreviousMajorVersions.spec.js

```diff
@@ -0,0 +1,62 @@
+const child = require('child_process');
+const semver = require('semver');
+const Dgeni = require('dgeni');
+const basePackage = require('../index');
+
+describe('getPreviousMajorVersions', function() {
+  let getPreviousMajorVersions;
+
+  beforeEach(function() {
+    const mockPackage = new Dgeni.Package('mock-package', [basePackage])
+                            .factory('versionInfo', mockVersionInfo)
+                            .factory('packageInfo', mockPackageInfo);
+    const dgeni = new Dgeni([mockPackage]);
+    const injector = dgeni.configureInjector();
+    getPreviousMajorVersions = injector.get('getPreviousMajorVersions');
+  });
+
+  it('should span a child process to git', function() {
+    spyOn(child, 'spawnSync').and.returnValue({status: 0, stdout: ''});
+    getPreviousMajorVersions();
+    expect(child.spawnSync).toHaveBeenCalledWith('git', ['ls-remote', '--tags', 'SOME_GIT_URL'], {
+      encoding: 'utf8'
+    });
+  });
+
+  it('should return an empty list for a failed git command', function() {
+    spyOn(child, 'spawnSync').and.returnValue({status: 1});
+    expect(getPreviousMajorVersions()).toEqual([]);
+  });
+
+  it('should return an empty list for no tags', function() {
+    spyOn(child, 'spawnSync').and.returnValue({});
```

Comment:
This is again testing the `status !== 0` case. I believe we need:

```suggestion
    spyOn(child, 'spawnSync').and.returnValue({status: 0, stdout: ''});
```